### PR TITLE
Display draft cards

### PR DESCRIPTION
**Pull request by ThomasSupervil**
Add the display of the 20 random cards from the API

### CODE REVIEW

**Review by CaroD59 — DISMISSED** [review]

**Review by CaroD59 — DISMISSED** [review]
Ok ! :)

File: src/components/Draft/Cards.jsx [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+
+  const [selectedHero, setSelectedHero] = useState(sampleCharacter);
+  const [availableHeroes, setAvailableHeroes] = useState([]);
+
+  useEffect(() => {
+    const heroes = [];
+    for (let i = 0; i < 20; i += 1) {
+      const randomID = Math.floor(Math.random() * 730);
+      axios
+        .get(`https://superheroapi.com/api.php/4118415658211107/${randomID}`)
```
Comment (LoicBrassart) [inline src/components/Draft/Cards.jsx]: It would be better to store the API key in a `.env` file
(Not seen yet)

**Review by LoicBrassart — DISMISSED** [review]
No real Evil, good job!

**Review by CaroD59 — DISMISSED** [review]
Ok for me !

**Review by CaroD59 — DISMISSED** [review]
Ok for me !

File: public/reset.css [hunk 2]
```diff
@@ -0,0 +1,129 @@
+/* http://meyerweb.com/eric/tools/css/reset/
```
Comment (LoicBrassart) [inline public/reset.css]: This file has nothing to do here ? 
It should be in `src` (and there **is** another copy in `src`!)

File: src/style.css [hunk 3]
```diff
@@ -1,17 +1,3 @@
...
-  margin-left: 100px;
-  display: flex;
-  flex-direction: row;
-  flex-wrap: wrap;
-}
 
-.perso img {
-  height: 155px;
-}
+
```
Comment (LoicBrassart) [inline src/style.css]: What's this ? An empty file ? Delete it

**Review by LoicBrassart — CHANGES_REQUESTED** [review]
Some file cleanup to do

File: src/style.css (hunk 3 above)
Comment (Sofiaboud) [reply src/style.css]: done

**Comment by LoicBrassart** [comment]
- SM validation before assigning me (frustrating, uh ? :p )
- Resolve conflicts before assigning me

**Review by CaroD59 — APPROVED** [review]
Ok for me :)

**Review by LoicBrassart — APPROVED** [review]
No more Evil

**Pull request merged**